packages/ckeditor5-dev-tests/lib/utils/transformfileoptiontotestglob.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/**
+ * Converts values of --files argument to proper globs.
+ * There are 5 supported types of values now:
+ *
+ * 0. current package's tests (when run in context of a package – e.g. on CI) - '/'
+ * 1. all packages' files – '*'
+ * 2. given package files – 'engine'
+ * 3. everything except the given package – '!engine'
+ * 4. path – 'engine/view' -> 'ckeditor5-engine/tests/view/**\/*.js'
+ * 5. simplified glob – 'engine/view/**\/*.js' -> 'ckeditor5-engine/tests/view/**\/*.js'
+ *
+ * @param {String} fileOption A path or pattern to determine the tests to execute.
+ * @param {Boolean} [isManualTest=false] Whether the tests are manual or automated.
+ * @returns {String}
+ */
+module.exports = function transformFileOptionToTestGlob( fileOption, isManualTest = false ) {
```

Comment:
For the future – such flags are hard to understand. When you see a call `transformFileOptionToTestGlob( '*', true )` you have no idea what that `true` is. So it's better to make it a string flag for instance.